ipynb/observationTable.md

```diff
@@ -0,0 +1,164 @@
+
+
+```python
+import pandas as pd
+import os.path
+```
+
+
+```python
+path_root = '/media/barnacle/projects/emp/'
+resultsFile = '/home/sjanssen/Desktop/observations.tsv'
```

Comment:
Right, but if there enough of these paths, it can confusing how to run the notebook.

There are a couple of ways around this
1. Use relative paths instead of absolute paths.  That way, it only makes assumptions about the project file structure.  This is what I do in all of my analyses, since I have standardized all of my projects.
2. Have a configuration file.  This is what the [American Gut repo](https://github.com/biocore/American-Gut) does

I think approach 1 would be just fine.